comdb2/dbapi2.py

```diff
@@ -240,24 +240,52 @@
    cursor for that connection.
 """
 
+from __future__ import annotations
+
```

Comment:
```suggestion
```

Same

Reply:
As above, we'd need to quote some annotations because of forward declarations, and we wouldn't be able to use `| None` for unions, or `list[str]` for lists, etc...